<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Aula</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
            <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                    <widget class="QComboBox" name="comboInscricao"/>
                </item>
                <item row="1" column="0">
                    <widget class="QDateEdit" name="dateEdit">
                        <property name="displayFormat">
                            <string>yyyy-MM-dd</string>
                        </property>
                    </widget>
                </item>
                <item row="2" column="0">
                    <widget class="QTimeEdit" name="timeEdit"/>
                </item>
                <item row="3" column="0">
                    <widget class="QPushButton" name="btnSalvar">
                        <property name="text">
                            <string>Salvar</string>
                        </property>
                    </widget>
                </item>
                <item row="3" column="1">
                    <widget class="QPushButton" name="btnCancelar">
                        <property name="text">
                            <string>Cancelar</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>